.gitignore

```diff
@@ -4,3 +4,5 @@ resources/
 .vercel
 *.lock
 hugo_stats.json
+.DS_Store
+.vscode/settings.json
```

Comment:
i'd personally just ignore the whole vscode folder

Reply:
Normally yes, but this repo has a few shared VS Code scripts ("tasks", see the [setup.md](https://github.com/mastodon/blog/blob/master/setup.md#vs-code-helpers)) in that folder that I wanted to preserve.